### ISSUE TITLE
Unexpected compression encoding for release archives

### ISSUE THREAD

**Issue by aviaviavi** [PostgREST/postgrest#1397]
### Environment

* PostgreSQL version: NA
* PostgREST version: 6.0.2
* Operating system: MacOS Mojave

### Description of issue

When pulling down the release tarballs for 6.0.2, they are a `.xz` file, but from what I can tell they appear to be bzip compressed. 

```shell
$ file postgrest-v6.0.2-osx.tar.xz
postgrest-v6.0.2-osx.tar.xz: bzip2 compressed data, block size = 900k
```

It's not a huge deal, just made pulling down this binary from within some of my Haskell code a bit tricky :). 

I think at the very least, the compression encoding should match the file extension, and also maybe a more common one like gzip could be used?

**Comment by steve-chavez** [comment]
That's true for the OSX release. Sorry about that.

Here I used bzip(`tar cjf`) instead of xz (`tar cJf`):
https://github.com/PostgREST/postgrest/blob/80f763448f9f84a7b783af79c05ddb92265fb31c/.travis.yml#L68

Already corrected `postgrest-v6.0.2-osx.tar.xz` on the releases page. Will fix the travis file.

**Comment by aviaviavi** [comment]
Thanks!
